doc/Pacemaker_Explained/en-US/Ch-Constraints.txt

```diff
@@ -208,21 +211,22 @@ indexterm:[Resource,Constraints,Ordering]
 indexterm:[Resource,Start Order]
 indexterm:[Ordering Constraints]
 
-The way to specify the order in which resources should start is by
-creating +rsc_order+ constraints.
+'Ordering constraints' tell the cluster the order in which resources should
+start.
 
 [IMPORTANT]
 ====
 Ordering constraints affect 'only' the ordering of resources;
-they do not require that the resources be placed on the
+they do 'not' require that the resources be placed on the
 same node. If you want resources to be started on the same node
-'and' in a specific order, you need an ordering constraint 'and'
-a location constraint (see <<s-resource-colocation>>).
+'and' in a specific order, you need both an ordering constraint 'and'
+a colocation constraint (see <<s-resource-colocation>>), or
+alternatively, a group (see <<group-resources>>).
```

Comment:
One day, in all our free time, we should create a "needs" (or some better name) constraint which implies ordering and colocation.